.travis.yml

```diff
@@ -8,7 +8,8 @@ before_install:
   - sudo add-apt-repository --yes ppa:ubuntu-toolchain-r/test
   #new cairo
   - sudo add-apt-repository --yes ppa:ricotz/testing
-
+  # new cmake
+  - sudo add-apt-repository --yes ppa:kalakris/cmake
```

Comment:
It's whitelisted as a dependency for the new container-based travis infrastructure. I'm quite sure there isn't a more official way until travis moves to Ubuntu 14.04 (whenever that will be). For the time being, I think this is fine.